src/sentry/integrations/vsts/integration.py

```diff
@@ -136,6 +137,44 @@ def get_organization_config(self):
             },
         ]
 
+    def update_organization_config(self, data):
+        if 'sync_status_forward' in data:
+            project_ids_and_statuses = data.pop('sync_status_forward')
+            data['sync_status_forward'] = bool(project_ids_and_statuses)
+
+            project_ids = project_ids_and_statuses.keys()
+            all_external_projects = IntegrationExternalProject.objects.filter(
+                organization_integration_id=self.org_integration.id,
+            )
+            all_external_projects.exclude(
+                external_id__in=project_ids,
+            ).delete()
+
+            existing_external_projects = all_external_projects.filter(
+                external_id__in=project_ids,
+            ).values_list('external_id', flat=True)
+
+            for project_id, statuses in project_ids_and_statuses.items():
+                if six.text_type(project_id) in existing_external_projects:
+                    IntegrationExternalProject.objects.get(
+                        external_id=project_id,
+                        organization_integration_id=self.org_integration.id,
+                    ).update(
```

Comment:
generally if you're doing an update like this, you want to use `.filter` rather than `.get` because the `.get` query executes immediately, which will result in two queries, whereas the filter + update will perform one update query